app/src/ui/lib/list/list-row.tsx

```diff
@@ -115,13 +131,27 @@ export class ListRow extends React.Component<IListRowProps, {}> {
 
   private onRowKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
     this.props.onRowKeyDown(this.props.rowIndex, e)
+    this.keyboardFocusDetectionState = 'failed'
+  }
+
+  private onRowKeyUp = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    this.props.onRowKeyDown(this.props.rowIndex, e)
```

Comment:
Shouldn't this be calling `this.props.onRowKeyUp`?

Reply:
Yep, you're right 😭  Nice catch!